README.md

```diff
@@ -18,51 +18,82 @@ See a walkthought video about it here :
 
 [![Pipelines as Code Walkthought](https://img.youtube.com/vi/Uh1YhOGPOes/0.jpg)](https://www.youtube.com/watch?v=Uh1YhOGPOes)
 
-## Installation
 
-Please follow [this document](INSTALL.md) for the install process
+Pipeline as Code features:
 
-## Features
+- Pull-request status support: When iterating over a Pull Request, status and control is
+  done on the platform.
 
-- Repository CRD: The Repository CRD is a new API introduced in the Pipelines as
-  Code project. This CRD is used to define the association between the source
-  code repository and the Kubernetes namespace in which the corresponding
-  Pipelines are run. It stores the status of the last runs as well.
+- GitHub Checks API support to set the status of a PipelineRun including rechecks
 
-- Web VCS support. When iterating over a Pull Request, status and control is
-  done on the platform.
+- GitHub Pull Request and Commit event support
 
-  *GitHub*:
+- Pull-request actions in comments such as `/retest`
 
-  - Support for Checks API to set the status of a PipelineRun.
-  - Support rechecks on UI.
-  - Support for Pull Request events.
-  - Use GitHUB blobs and objects API to get configuration files directly.
-    (instead of checking the repo locally)
+- Git events filtering and support for separate pipelines for each event
 
-* Remote tasks
+- Automatic Task resolution in Pipelines (local Tasks, Tekton Hub and remote URLs)
 
-  - Automatically grabs remote tasks from Tekton HUB or from Remote URI
+- Efficient use of GitHub blobs and objects API for retrieving configurations
 
-* CLI - tkn-pac
+- `tkn-pac` plugin for Tekton CLI for managing pipeline-as-code repositories and bootstrapping
+  
 
-  - A CLI to let you list and describe your repositories and runs attached to it.
-  - Quickly bootstrap a pipelinerun and repository CRD.
+## Installation Guide
 
-## User flow
+Please follow [this document](INSTALL.md) for installing Pipeline as Code on OpenShift.
 
-### GitHub apps Configuration
+## Getting Started
 
-- Admin gives the GitHub application url to add to the user.
-- User clicks on it and add the app on her repository which is in this example
-  named `linda/project`
-- Users create a namespace inside their kubernetes where the runs are going to
-  be executed. i.e:
+The flow for using pipeline as code generally begins with admin installing the Pipeline-as-Code infrastructure, creating a GitHub App and sharing the GitHub App url across the organization for app teams to enable the app on their GitHub repositories.
+
+In order to enable the GitHub App provided by admin on your Git repository. Otherwise you can go to the *Settings > Applications* and then click on *Configure* button near the GitHub App you had created. In the **Repository access** section, select the repositories that you want to enable and have access to pipeline-as-code.
```

Comment:
```suggestion
In order to enable the GitHub App provided by admin on your Git repository as documented [here](https://docs.github.com/en/developers/apps/managing-github-apps/installing-github-apps). Otherwise you can go to the *Settings > Applications* and then click on *Configure* button near the GitHub App you had created. In the **Repository access** section, select the repositories that you want to enable and have access to pipeline-as-code.
```